packages/infrastructure/mongodb_atlas_identity_provider/provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "mongodbatlas" {
```

Comment:
@fullykubed not sure if this is an anti-pattern for us. is it ok that a `provider.tf` file exists like this?

Reply:
@fullykubed given this, what is the desired way of providing secrets like the private_key and public_key that I am providing through a `var` currently. Is the expectation that this secret is no longer managed in code but through env by each engineer interfacing with iac?

Reply:
See the authentik provider for what I'd recommend here.